docs/association/security/an_bug_bounty.md

```diff
@@ -0,0 +1,77 @@
+# Aragon Network smart contracts bug bounty
+
+A bug bounty for the smart contracts of the Aragon Network is now live. We intend for hackers to look for smart contract vulnerabilities in our system that can lead to loss of funds or locked components.
+
+## Rewards
+
+Vulnerability reports will be scored using the  [CVSS v3](https://www.first.org/cvss/) standard. The reward amounts for different types of vulnerabilities are:
+
+!!! bug "**Critical** (CVSS 9.0 - 10.0)"
+    $5,000 - $50,000
+
+!!! danger "**Major** (CVSS 7.0 - 8.9)"
+    $2,500 - $5,000
+
+!!! fail "**Medium** (CVSS 4.0 - 6.0)"
+    $1,000 - $2,500
+
+!!! warning "**Low** (CVSS 1.0 - 3.9)"
+    $500-$1,000
+
+Rewards will be awarded at the sole discretion of the Aragon Association. Quality of the report and reproduction instructions can impact the reward. Rewards will be paid out in ETH.
+
+For this initial bug bounty program, there is a **maximum bounty pool of $250,000**.
+
+The bug bounty program is ongoing and has been running since November 25th, 2019.
+
+## Reporting
+
+!!! note "Reporting a found vulnerability"
+    Please responsibly disclose any findings to the development team. You may find instructions on how to do so in the [security overview page](../).
+
+    Failure to do so will result in a finding being ineligible for any bounties.
+
+## Scope
+
+In scope for the bug bounty are all the smart contract components of the Aragon Network. They can be found in the following repositories:
+
+!!! abstract "**Aragon Court** ([v1](https://github.com/aragon/aragon-court/tree/v1.0.0) and [future minor versions](https://github.com/aragon/aragon-court/releases))"
+    Dispute resolution protocol.
+
+    - Solidity code under the `contracts` directory:
+        - Excluding `contracts/lib/os/` 
+        - Excluding `contracts/test/`
+        - Excluding `contracts/standards/`
+
+!!! abstract "**Aragon Network deployment** ([v1](https://github.com/aragon/aragon-network-deploy/tree/v1.0.0) and [future minor versions](https://github.com/aragon/aragon-network-deploy/releases))"
```

Comment:
Not sure about adding this to the bug bounty; maybe we should just update this with the actually deployed addresses once those are ready?

Similar to the Aragon client bug bounty:

> You can find their deployed addresses on live networks in our deployment documentation.

Reply:
Sure we can add them once ready, however, it is not harmful to add the deployment scripts, what do you think? IMO it is a critical part of the project too

Reply:
Sounds good 👍; it'll be useful to add the final deployment addresses to the deployment scripts as well.